packages/kbn-unified-field-list/src/hooks/use_field_filters.ts

```diff
@@ -75,7 +75,10 @@ export function useFieldFilters<T extends FieldListItem = DataViewField>({
       onFilterField:
         fieldSearchHighlight?.length || selectedFieldTypes.length > 0
           ? (field: T) => {
-              if (fieldSearchHighlight && !fieldNameWildcardMatcher(field, fieldSearchHighlight)) {
+              if (
+                fieldSearchHighlight?.trim() &&
```

Comment:
is there a case where the untrimmed version is needed ? else doing the trim e.g. here:

https://github.com/elastic/kibana/blob/4f9e2bdf195caace9ce860230929c86e31cacb6a/packages/kbn-unified-field-list/src/hooks/use_field_filters.ts#L61

could make some follow up trims redundant ... just a thought, nothing blocking

Reply:
@kertal True, updated it in this file. I think it's better to keep another trimming in `fieldNameWildcardMatcher` since the logic relies on it.